packages/cli/src/controller/init-controller.test.ts

```diff
@@ -67,7 +67,7 @@ describe('Cli can create project', () => {
   it('prepare correctly applies project details', async () => {
     const tempPath = await makeTempDir();
     const templates = await fetchTemplates();
-    const template = templates.find(({name}) => name === 'polkadot-starter');
+    const template = templates.find(({name}) => name === 'Polkadot-starter');
```

Comment:
Why does this keep changing? It was changed not that long ago. Maybe we should just do a case insensitive check